pages/boards.tsx

```diff
@@ -1,19 +1,21 @@
 import BestBoards from "@/components/boards/BestBoards";
 import BoardList, { BoardListProps } from "@/components/boards/BoardList";
+import { ARTICLE_URL } from "@/constants/url";
 import { fetchData } from "@/lib/fetchData";
+import { ArticleProps } from "@/types/articleTypes";
 import { GetServerSidePropsContext } from "next";
-import { useAuth } from "@/contexts/AuthProvider";
-import { useEffect } from "react";
 
 export const getServerSideProps = async (
   context: GetServerSidePropsContext
 ) => {
   const keyword = context.query["keyword"] ?? "";
 
-  const BASE_URL = "https://panda-market-api.vercel.app/articles";
-  const { list } = await fetchData(BASE_URL, {
-    query: { keyword: typeof keyword === "string" ? keyword : "" },
-  });
+  const { list } = await fetchData<Record<string, ArticleProps[]>>(
```

Comment:
P1:
지금 선언해주신 `Record<string, ArticleProps[]>` 타입이 의미하는 것은 객체의 키가 string 타입이고, 그 값은 ArticleProps 타입을 가진 배열이라는 뜻입니다.
의도하신 바는 아마 `list` 키값에 `ArticleProps[]` 타입을 가진 배열이 응답값으로 내려온다는 것이므로 아래 코드가 더 적절해보입니다.

```suggestion
  const { list } = await fetchData<{ list: ArticleProps[] }>(
```